launcher/jvm_tooling.h

```diff
@@ -25,6 +25,15 @@ extern std::string FLAGS_jvm_args;
 extern std::string FLAGS_additional_jvm_args;
 extern std::string FLAGS_agent_path;
 
+#define ARG_SEPARATOR ","
```

Comment:
Isn't this a breaking change? Do we have to update all OSS-Fuzz build scripts? 

I would prefer to not make any sweeping changes like this to the native launcher as we are only keeping it around for backwards compatibility anyway.

Reply:
same here. Dropped the commit related to changing the argument separator in the launcher